components/core/src/clp/Utils.cpp

```diff
@@ -262,38 +265,29 @@ void load_lexer_from_file(
         }
 
         if (contains_delimiter) {
-            FileReader schema_reader;
-            ErrorCode error_code = schema_reader.try_open(schema_ast->m_file_path);
-            if (ErrorCode_Success != error_code) {
```

Comment:
The exception throw here doesn't make sense. actually it would be hard to imagine that the file was opened once but failed to be opened again. so I would rather leave it to throw the default exception from File_reader